<ui version="4.0" >
 <class>SChatWindow</class>
 <widget class="QMainWindow" name="SChatWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Simple Chat</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>581</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2" >
    <property name="margin" >
     <number>6</number>
    </property>
    <item row="0" column="0" >
     <layout class="QVBoxLayout" name="verticalLayout_2" >
      <property name="topMargin" >
       <number>0</number>
      </property>
      <item>
       <widget class="QSplitter" name="splitter" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QTabWidget" name="tabWidget" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Expanding" hsizetype="Minimum" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="tabPosition" >
          <enum>QTabWidget::North</enum>
         </property>
         <property name="tabShape" >
          <enum>QTabWidget::Rounded</enum>
         </property>
         <property name="currentIndex" >
          <number>0</number>
         </property>
         <property name="elideMode" >
          <enum>Qt::ElideLeft</enum>
         </property>
         <widget class="QWidget" name="tab" >
          <property name="geometry" >
           <rect>
            <x>0</x>
            <y>0</y>
            <width>388</width>
            <height>254</height>
           </rect>
          </property>
          <attribute name="title" >
           <string>Tab 1</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout" >
           <property name="margin" >
            <number>0</number>
           </property>
           <item row="0" column="0" >
            <widget class="QTextBrowser" name="textBrowser" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="focusPolicy" >
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="frameShape" >
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow" >
              <enum>QFrame::Sunken</enum>
             </property>
             <property name="lineWidth" >
              <number>1</number>
             </property>
             <property name="openExternalLinks" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2" >
          <property name="geometry" >
           <rect>
            <x>0</x>
            <y>0</y>
            <width>388</width>
            <height>254</height>
           </rect>
          </property>
          <attribute name="title" >
           <string>Tab 2</string>
          </attribute>
         </widget>
         <widget class="QWidget" name="tab_3" >
          <property name="geometry" >
           <rect>
            <x>0</x>
            <y>0</y>
            <width>388</width>
            <height>254</height>
           </rect>
          </property>
          <attribute name="title" >
           <string>Page</string>
          </attribute>
         </widget>
         <widget class="QWidget" name="tab_4" >
          <property name="geometry" >
           <rect>
            <x>0</x>
            <y>0</y>
            <width>388</width>
            <height>254</height>
           </rect>
          </property>
          <attribute name="title" >
           <string>Page</string>
          </attribute>
         </widget>
         <widget class="QWidget" name="tab_5" >
          <property name="geometry" >
           <rect>
            <x>0</x>
            <y>0</y>
            <width>388</width>
            <height>254</height>
           </rect>
          </property>
          <attribute name="title" >
           <string>Page</string>
          </attribute>
         </widget>
        </widget>
        <widget class="QWidget" name="layoutWidget" >
         <layout class="QVBoxLayout" name="verticalLayout" >
          <property name="rightMargin" >
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="lineEdit_2" >
            <property name="text" >
             <string>Newbie</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListView" name="listView" >
            <property name="focusPolicy" >
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="editTriggers" >
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="movement" >
             <enum>QListView::Static</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" >
        <item>
         <widget class="QLineEdit" name="lineEdit" />
        </item>
        <item>
         <widget class="QPushButton" name="pushButton" >
          <property name="text" >
           <string>Подключится</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>581</y>
     <width>800</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <action name="actionDsd" >
   <property name="text" >
    <string>dsd</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>lineEdit_2</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
